src/render.spec.ts

```diff
@@ -0,0 +1,47 @@
+import expect from "expect";
+import { getRenderedGameState } from "./render";
+import { GameState, CellValue } from "./gamestate";
+
+describe("Draw a boad from a game state object", () => {
+  it("Draw 2-rows board with all cells empty", () => {
+    const twoRowsAllCellsEmpty: GameState = {
+      board: [
+        [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+        [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+      ],
+    };
+    expect(getRenderedGameState(twoRowsAllCellsEmpty)).toBe("⬡⬡\n ⬡⬡\n");
```

Comment:
Use the Given/When/The pattern in order to ease the test readability

Reply:
Am i using it correctly ?

```
const emptyBoard = {
      board: [],
};

const renderedGameState = getRenderedGameState(emptyBoard);

expect(renderedGameState).toBe("");
```